modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -265,18 +278,20 @@ export default class LargeVideoManager {
      * shown on the large video area.
      *
      * @param {string} id the id of remote participant(MUC nickname)
-     * @param {boolean} isConnected true if the connection is active or false
-     * when the user is having connectivity issues.
+     * @param {boolean} showProblemsIndication
+     * @param {boolean} showMessage
```

Comment:
Is the change in signature here necessary for clarity or could the existence of messageKey also be used as the showMessage check?